jm-glass/src/components/Tableture.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { StringedInstrument } from '@engine/strings';
+
+const LINE_HEIGHT = 30;
+
+const Tableture: React.FC<{
+  instrument: StringedInstrument;
+  tab: number[][];
+  clickHandler: (index: number) => void;
+  selected: number[];
+}> = ({ instrument, tab, clickHandler, selected }) => {
+  return (
+    <div style={{ display: 'inline' }}>
+      <TabStartSVG instrument={instrument} />
+      {tab.map((chord, index) => {
+        return (
+          <TabSVG
+            chord={chord}
+            index={index}
+            instrument={instrument}
+            key={index}
+            handleClick={clickHandler}
+            selected={selected}
+          />
+        );
+      })}
+      <TabEndSVG instrument={instrument} />
+    </div>
+  );
+};
+
+const TabSVG: React.FC<{
+  chord: number[];
+  instrument: StringedInstrument;
+  index: number;
+  handleClick: (index: number) => void;
+  selected: number[];
+}> = ({ chord, instrument, index, handleClick, selected }) => {
+  const backgroundColor = selected.includes(index) ? 'black' : 'white';
+  const lineStrokeColor = selected.includes(index) ? 'white' : 'black';
+  const numberStrokeColor = selected.includes(index) ? 'white' : 'black';
+  return (
+    <svg
+      style={{ display: 'inline', backgroundColor: `${backgroundColor}` }}
```

Comment:
i think it can just be 
`{{ display: 'inline', backgroundColor }}`

Reply:
Done! Thanks @krasovsky22